src/Orleans/Runtime/RequestContext.cs

```diff
@@ -42,6 +41,11 @@ public static class RequestContext
         internal const string ORLEANS_REQUEST_CONTEXT_KEY = "#ORL_RC";
         internal const string PING_APPLICATION_HEADER = "Ping";
 
+#if NETSTANDARD_TODO
```

Comment:
Oh, I see. Just wanted to mark it as subject for further complete removal of <code>NETSTANDARD</code> condition. Will rename.